addons/packages/vsphere-cpi/1.23.0/test/vsphere_cpi_test.go

```diff
@@ -682,6 +757,15 @@ func parseDaemonSet(output string) appsv1.DaemonSet {
 	return daemonSet
 }
 
+func parseDeployment(output string) appsv1.Deployment {
+	deploymentDocIndex := 8
```

Comment:
This gonna bite us in the future.. anyway

Reply:
hahaha